src/Makefile

```
@@ -14,22 +14,49 @@ ifneq ($(ENABLE_TMS), 1)
 endif
 
 
-# Automatically enable SAND if SANDRECO_INC or SANDRECO_LIB are set
-ifneq ($(origin SANDRECO_INC), undefined)
-	INCLUDE += -I$(SANDRECO_INC)
-	SAND_OPT = -DENABLE_SAND
-endif
+# # Automatically enable SAND if SANDRECO_INC or SANDRECO_LIB are set
+# ifneq ($(origin SANDRECO_INC), undefined)
+# 	INCLUDE += -I$(SANDRECO_INC)
+# 	SAND_OPT = -DENABLE_SAND
+# endif
 
-ifneq ($(origin SANDRECO_LIB), undefined)
-	LDLIBS += -L$(SANDRECO_LIB) -lStruct
-	SAND_OPT = -DENABLE_SAND
-endif
+# ifneq ($(origin SANDRECO_LIB), undefined)
+# 	LDLIBS += -L$(SANDRECO_LIB) -lStruct
+# 	SAND_OPT = -DENABLE_SAND
+# endif
 
 
 LIBNAME = ND_CAFMaker
 LIBS = $(LIBDIR)/lib$(LIBNAME).so
 BINS = $(BINDIR)/makeCAF
 
+# Option to enable struct.h dictionary (default: off)
+ENABLE_SAND_DICT ?= 0
+
+# Add variables for struct dictionary and library
+DEPSDIR = $(SRCDIR)/deps
+STRUCT_HDR = $(DEPSDIR)/struct.h
+STRUCT_LINKDEF = $(DEPSDIR)/StructLinkDef.h
+STRUCT_DICT_SRC = $(LIBDIR)/StructDict.cxx
+STRUCT_DICT_OBJ = $(LIBDIR)/StructDict.o
+STRUCT_LIB = $(LIBDIR)/libStruct.so
+STRUCT_PCM = $(LIBDIR)/libStruct_rdict.pcm
+
+ifeq ($(ENABLE_SAND_DICT),1)
+$(shell wget https://raw.githubusercontent.com/DUNE/sandreco/refs/heads/develop/include/struct.h -P $(DEPSDIR))
+$(shell wget https://raw.githubusercontent.com/DUNE/sandreco/refs/heads/develop/include/StructLinkDef.h -P $(DEPSDIR))
+INCLUDE += -I$(DEPSDIR)
+SAND_OPT = -DENABLE_SAND
+LDLIBS_EXEC += -L$(LIBDIR) -lStruct
+DICT_TARGETS = $(STRUCT_LIB)
+else
+DICT_TARGETS =
+LDLIBS_EXEC =
+SAND_OPT =
+endif
```

can these please be indented for readability?
```suggestion
ifeq ($(ENABLE_SAND_DICT),1)
	$(shell wget https://raw.githubusercontent.com/DUNE/sandreco/refs/heads/develop/include/struct.h -P $(DEPSDIR))
	$(shell wget https://raw.githubusercontent.com/DUNE/sandreco/refs/heads/develop/include/StructLinkDef.h -P $(DEPSDIR))
	INCLUDE += -I$(DEPSDIR)
	SAND_OPT = -DENABLE_SAND
	LDLIBS_EXEC += -L$(LIBDIR) -lStruct
	DICT_TARGETS = $(STRUCT_LIB)
else
	DICT_TARGETS =
	LDLIBS_EXEC =
	SAND_OPT =
endif
```